README.md

```diff
@@ -25,6 +25,8 @@ the Google Play Store from a continuous integration server or anywhere you have
    1. [Using multiple Service Accounts](#using-multiple-service-accounts)
    1. [Encrypting Service Account keys](#encrypting-service-account-keys)
    1. [Running custom tasks before publishing](#running-custom-tasks-before-publishing)
+1. [Advanced](#advanced)
+   1. [Using Alternate Build Folder](#alternate-build-folder)
```

Comment:
`Using alternate build folders` with the `s` and no caps since we're using sentence case